shared/fileutil/fileutil_test.go

```diff
@@ -13,13 +13,22 @@
 //
 // You should have received a copy of the GNU General Public License
 // along with go-ethereum. If not, see <http://www.gnu.org/licenses/>.
-package fileutil
+package fileutil_test
 
 import (
+	"bufio"
+	"bytes"
+	"io/ioutil"
+	"log"
 	"os"
 	"os/user"
 	"testing"
 
+	"github.com/prysmaticlabs/prysm/shared/params"
+
+	"github.com/prysmaticlabs/prysm/shared/fileutil"
+
+	"github.com/prysmaticlabs/prysm/shared/testutil"
```

Comment:
```suggestion
	"github.com/prysmaticlabs/prysm/shared/params"
	"github.com/prysmaticlabs/prysm/shared/fileutil"
	"github.com/prysmaticlabs/prysm/shared/testutil"
```
You might need to run goimports to sort these